scripts/Makefile-gaf-reprocess

```diff
@@ -1,9 +1,22 @@
 BASE_NAMES = $(notdir $(wildcard /opt/go-site/annotations/*))
 
-all: $(foreach name,$(BASE_NAMES),/opt/go-site/annotations_new/$(name))
+all: $(foreach name,$(BASE_NAMES),/opt/go-site/annotations_new/$(name)) /opt/go-site/gaferencer-products/all.gaferences.json.gz
 
-/opt/go-site/annotations_new/%.gaf: /opt/go-site/annotations/%.gaf
+/opt/go-site/annotations_new/%.gz: /opt/go-site/annotations_new/%
+	pigz /opt/go-site/annotations_new/$*
```

Comment:
I'd recommend using `$<` maybe instead of trying to reconstruct the path, when you already have it in the prereq.